model_security_adjust_cft/security/ir.model.access.csv

```diff
@@ -5,3 +5,6 @@ account.access_account_invoice_group_invoice,True,account.invoice group invoice,
 account.access_account_invoice_accountant,True,account.invoice accountant,account.model_account_invoice,account.group_account_user,True,True,True,True
 stock.access_stock_picking_user,True,stock.picking user,stock.model_stock_picking,stock.group_stock_user,True,FALSE,FALSE,FALSE
 stock.access_stock_move_user,True,stock.move user,stock.model_stock_move,stock.group_stock_user,True,FALSE,FALSE,FALSE
+stock.access_product_template_stock_user,True,product.template stock user,account.model_product_template,stock.group_stock_user,True,FALSE,FALSE,FALSE
+stock.access_product_template_stock_manager,True,product.template stock_manager,account.model_product_template,stock.group_stock_manager,True,FALSE,FALSE,FALSE
+purchase.access_product_template_purchase_user,True,product.template purchase_user,account.model_product_template,purchase.group_purchase_user,True,True,True,True

```

Comment:
Use `product.model_product_template` instead of `account.model_product_template` for these lines you added.